openseeds/cohort-procedures-and-templates/cohort-call-facilitator-guide.md

```diff
@@ -10,108 +10,121 @@ photos:
 ---
 
 
-We are very excited to create this role in OLS for the returning members from the previous cohorts of OLS.
+We are very excited to create these roles in OLS for returning members from the previous cohorts of OLS.
 
-We want to invite support from our community members in making the cohorts successful.
+We want to invite support from our community members to help make the cohorts successful.
 
 At the same time, our hope is to share our practices for training and planning so you can take them into your projects and community.
 
-Our cohort call facilitators share some of the cohort call-related management roles with the OLS team. 
+Facilitators share some of the cohort call-related management roles with the OLS team. 
 
-Please identify which week you have signed up for and who is the cohort call lead.
+Please identify which week you have signed up for and who the cohort call lead is.
 
-Our cohort calls vary in time from week to week. Please, see schedules from [our website]({{ site.url }}/openseeds/).
+Our cohort calls vary in time from week to week. Please, see the schedules from [our website]({{ site.url }}/openseeds/).
 
-For any questions or concerns, please contact Community Coordinator, Paz Bernaldo via Slack.
+For any questions or concerns, please contact the cohort coordinators via Slack or email.
```

Comment:
Who are they, @iramosp?

Reply:
Might we worth mentioning the key players. What do you think?

Reply:
Good idea, I added contact email for coordinators.